src/WebAssert.php

```diff
@@ -565,6 +565,32 @@ public function elementAttributeExists($selectorType, $selector, $attribute)
         return $element;
     }
 
+    /**
+     * Checks that an attribute does not exist in an element.
+     *
+     * @param string       $selectorType
+     * @param string|array $selector
+     * @param string       $attribute
+     *
+     * @return NodeElement
+     *
+     * @throws ElementHtmlException|ElementNotFoundException
```

Comment:
Please remove `|ElementNotFoundException` that you've added (when comparing with test for an `elementAttributeExists` method).

I've noticed, that none of the assertion methods using `$this->elementExists(...)` are telling that they're throwing an `ElementNotFoundException`.

Reply:
Oh, my bad, removed!

Reply:
Don't you think we should map this exception in tests? I mean, not now because this is out of the scope of this issue, but in another issue. Since the function can throw this exception.

Reply:
> Don't you think we should map this exception in tests? I mean, not now because this is out of the scope of this issue, but in another issue. Since the function can throw this exception.

No. Testing of assertion method IMO should test both passed and failed assertion and nothing more. It's out of the scope of assertion method testing to verify the internal logic of methods used inside it (e.g. for finding an element on the page). Also, it's likely not documented (that exception would be thrown on a missing element).

Reply:
Thanks a lot for this response, it makes sense!